docs/gateway/oss.md

```diff
@@ -1,4 +1,7 @@
 # MinIO OSS Gateway [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io)
+
+DEPRECATED: THE GATEWAY SUPPORT OF OSS IS NO LONGER MAINTAINED AND MAY NO LONGER WORK AS EXPECTED.
```

Comment:
You should simply update 

```go
// Production - oss is production ready.
func (g *OSS) Production() bool {
        return true
}
```

To return `false` in cmd/gateway/oss/gateway-oss.go  - this documention is not correct.

Reply:
That is a good point - I will change that as well. I would however still argue that it is also a good idea to state upfront whether a project is maintained any longer or not. By only changing the Boolean, it will be clear that the project-as-is is not production-ready when you run the container, but to find out that no further work will be done to the project, you will have to couple that with the project insights and my issue. By stating it in the documentation, the path to this rather important piece of information is shortened.

Reply:
I don't agree - someone may wish to fix it. Adding this line is not meaningful.

Reply:
I am perfectly fine with changing the wording of the line - as also written in my comment. Feel free to come up with any suggestion you would like. My point is only to have something also written in the documentation.

Reply:
P.S. I have now included your suggested code change in the PR.

Reply:
The code change is enough to indicate to not use in production and expect surprises.